### DISCUSSION TITLE
右上角最大化和向下还原是不是搞反了？

### DISCUSSION THREAD

**Discussion by Vatario** [qier222/YesPlayMusic#1399] [Q&A]
如题

**Comment by memorydream** [comment]
虽然不清楚你正在使用哪个版本，也没有图，不过应该是在 #1343 修复掉的问题，在其之前的版本中，windows版在启动时会是还原图标，最大化然后还原一次就变回正常了

**Reply by Vatario** [reply]
![image](https://user-images.githubusercontent.com/47292446/157621691-ecd4258e-db86-4ee9-adc2-b2b6fc2ff9fc.png)
![Q~{ 9O6FBJJ0IOSAROSOX 9](https://user-images.githubusercontent.com/47292446/157622347-49748c4c-9645-470b-a1e3-d4a8b08d7e6f.png)

**Reply by memorydream** [reply]
![image](https://user-images.githubusercontent.com/34763046/157622880-113bb26f-31eb-4ef4-87e2-0f7d0aaa4dec.png)
![image](https://user-images.githubusercontent.com/34763046/157622910-15cd431e-0d98-416a-a367-4d7894b56a9b.png)

**Reply by memorydream** [reply]
我有些迷惑，你正在使用哪个版本？通过什么方式来最大化/还原？

**Reply by Vatario** [reply]
![屏幕截图 2022-03-10 165013](https://user-images.githubusercontent.com/47292446/157625500-69a8bdb0-1860-49f6-9c8c-b9964ad4dedd.jpg)

**Reply by memorydream** [reply]
如果你正在使用release里下载的0.4.4-beta，确实存在 `最大化/还原` 按钮在特定情况下不正确的问题，因为相关的修复是在这个release之后的
https://github.com/qier222/YesPlayMusic/actions/runs/1945530715
这里有最新可用的构建，如果你很在意的话，可以下载使用，这个版本中已经修复了这个问题
